examples/binning/binning.py

```diff
@@ -7,7 +7,7 @@
 import pandas as pd
 from winton_kafka_streams.processor import BaseProcessor, TopologyBuilder
 import winton_kafka_streams.kafka_config as kafka_config
-import winton_kafka_streams.kafka_stream as kafka_stream
+import winton_kafka_streams.kafka_streams as kafka_streams
```

Comment:
Yes. But if the rename doesn't happen in this PR (and afaict, it doesn't), then is master working?